RatScanner/RatScanner.csproj

```diff
@@ -18,6 +18,9 @@
   </PropertyGroup>
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
     <DefineConstants>DEBUG;TRACE</DefineConstants>
+    <PlatformTarget>x64</PlatformTarget>
```

Comment:
These were part of getting it to run in VSCode, in theory these changes could be removed, but it still works in Visual Studio the same as always.